SWBF2Import/SWBF2Import.cs

```diff
@@ -151,6 +163,11 @@ public static class SWBF2Import {
         }
     }
 
+    public static string parseChars(String str)
```

Comment:
Maybe rename the method to sth that descibes what it does. Is public necessary?

Reply:
I see. IIRC grabbing the layer texture strings just performs a copy of the respective string block in the .ter file without any checks, which might contain empty bytes (00).

I wont though patch this in the C# version, but consider it in the C++ version (which should replace the C# version in the long term).